pkg/comp-functions/functions/common/common.go

```diff
@@ -29,7 +29,7 @@ func loadRuntimeFromFile(t assert.TestingT, file string) *runtime.Runtime {
 	return funcIO
 }
 
-func getFunctionIo(funcIO *runtime.Runtime) xfnv1alpha1.FunctionIO {
+func GetFunctionIo(funcIO *runtime.Runtime) xfnv1alpha1.FunctionIO {
 	field := reflect.ValueOf(funcIO).Elem().FieldByName("io")
 	return reflect.NewAt(field.Type(), unsafe.Pointer(field.UnsafeAddr())).Elem().Interface().(xfnv1alpha1.FunctionIO)
 }
```

Comment:
Shouldn't this file be `common_test.go`?

Reply:
From what we've just discussed offline:

Maybe put it in a separate package containing the word test? I don't like test functions to clutter the public API...

Reply:
Since test methods are not exported between packages I had to come up with a solution which I agree is not the best. I created another package `commontest` where I put the common functions for testing.